src/main/java/com/dougnoel/sentinel/steps/ImageVerificationSteps.java

```diff
@@ -104,23 +139,22 @@ private static ImageComparisonResult compareImages(String elementName) throws IO
 		int actualWidth = actualImage.getWidth();
 
 		if((expectedWidth != actualWidth) || (expectedHeight != actualHeight)) {
+			switch(pageType){
+				case WEBPAGE:
+					break;
+				case EXECUTABLE:
+				case UNKNOWN:
+				default:
+					log.warn("Page object has been detected as {} with different sized comparison snapshots. This may result in an unreliable comparison.", pageType);
+					break;
```

Comment:
It is unclear to me why the comparison wouldn't fail at this point for all types unless this is a "contains" instead of a "has".